include/sdf/Cylinder.hh

```diff
@@ -72,6 +75,9 @@ namespace sdf
     /// \return A reference to a gz::math::Cylinderd object.
     public: gz::math::Cylinderd &Shape();
 
+    /// \brief Calculate an return the Mass Matrix values for the Cylinder
+    public: std::optional< gz::math::MassMatrix3d >  MassMatrix(const double density);
```

Comment:
same comment as above

Reply:
Corrected in b092904db4ef5c491672619076e75f4509b3369b